testutil/test_app.go

```diff
@@ -177,26 +174,7 @@ func AddGenesisAccount(addr sdk.AccAddress, appState app.GenesisState, cdc codec
 	return appState, nil
 }
 
-func generateKeyring(t *testing.T, cdc codec.Codec, accts ...string) keyring.Keyring {
-	t.Helper()
-	kb := keyring.NewInMemory(cdc)
-
-	for _, acc := range accts {
-		_, _, err := kb.NewMnemonic(acc, keyring.English, "", "", hd.Secp256k1)
-		if err != nil {
-			t.Error(err)
-		}
-	}
-
-	_, err := kb.NewAccount(TestAccName, testMnemo, "1234", "", hd.Secp256k1)
-	if err != nil {
-		panic(err)
-	}
-
-	return kb
-}
-
-// SetupWithGenesisValSet initializes GenesisState with a single validator and genesis accounts
+// GenesisStateWithSingleValidator initializes GenesisState with a single validator and genesis accounts
```

Comment:
👍

Reply:
[no change needed] it would be awesome if we enabled a lint rule that flagged when the godoc for a function doesn't start with the name of the function. In the meantime, this type of correction is def appreciated!

Reply:
Yeah, this would be a nice addition 💯